src/extension/background-script/actions/allowances/update.js

```diff
@@ -2,11 +2,16 @@ import db from "../../db";
 
```

Comment:
You wanna try to switch this file to TS? If there's an issue we could pair on that.

Reply:
Will be done here:
https://github.com/getAlby/lightning-browser-extension/pull/1046